tests/lora/test_lora_manager.py

```diff
@@ -24,12 +24,19 @@ def test_from_lora_tensors(sql_lora_files):
         os.path.join(sql_lora_files, "adapter_model.safetensors"))
     new_embeddings = load_file(
         os.path.join(sql_lora_files, "new_embeddings.safetensors"))
-    lora_model = LoRAModel.from_lora_tensors(1,
-                                             8,
-                                             16,
-                                             tensors,
-                                             "cuda",
-                                             embeddings=new_embeddings)
+    embedding_modules = {
```

Comment:
It seems this is duplicating `EMBEDDING_MODULES`, let's reuse that here instead of defining a dict? Same goes for `{
            "embed_tokens": "input_embeddings",
            "lm_head": "output_embeddings",
        }`
below.